src/authorizations/components/CustomApiTokenOverlay.tsx

```diff
@@ -77,39 +77,54 @@ const CustomApiTokenOverlay: FC<Props> = props => {
   const [searchTerm, setSearchTerm] = useState('')
   const [status, setStatus] = useState(ComponentStatus.Disabled)
 
+  const {
+    allResources,
+    bucketPermissions,
+    getBuckets,
+    getTelegrafs,
+    remoteDataState,
+    telegrafPermissions,
+  } = props
+
   useEffect(() => {
-    props.getBuckets()
-    props.getTelegrafs()
-  }, [])
+    getBuckets()
+    getTelegrafs()
+  }, [getBuckets, getTelegrafs])
 
```

Comment:
Since these methods are being added to the dependency array - should we memoize the function with `useCallback`?

Reply:
I'm wrong here - there's nothing to memoize because this is being imported directly from another file (not passed from another component). The reason it's destructured from props is due to mdtp.